### ISSUE TITLE
Number of rows in object different from file

### ISSUE THREAD

**Issue by YichaoCaiDBS** [al2na/methylKit#133]
Hi, I've encountered a strange issue when I tried to load ENCODE RRBS data in using methRead().
The row number in methylRaw object is far less than what I have in my input file.

The input file can be access using:
```
wget http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeHaibMethylRrbs/wgEncodeHaibMethylRrbsK562HaibSitesRep1.bed.gz
zcat wgEncodeHaibMethylRrbsK562HaibSitesRep1.bed.gz | grep -v "^track" | gzip > k562_rep1.bed.gz
```

Then I tried to load this using methRead():
```
library(methylKit)
files <- list("k562_rep1.bed.gz")
MKobj <- methRead(files[[1]],
            sample.id = "K562_rep1",
            assembly = 'hg19',
            treatment = 0,
            header=FALSE,
            context = 'CpG',
            resolution = 'base',
            pipeline = list(fraction = FALSE,
                            chr.col = 1,
                            start.col = 3,
                            end.col = 3, 
                            coverage.col = 5,
                            strand.col = 6,
                            freqC.col = 11
            ))
# > Read 1275818 rows and 11 (of 11) columns from 0.090 GB file in 00:00:03
MKobj

methylRaw object with 808212 rows
--------------
   chr   start     end strand coverage numCs numTs
1 chr1 1000171 1000171      +       46    16    30
2 chr1 1000191 1000191      +       46     7    39
3 chr1 1000192 1000192      -       53     5    48
4 chr1 1000199 1000199      +       46     9    37
5 chr1 1000200 1000200      -       53     8    45
6 chr1 1000207 1000207      -       53    14    39
--------------
sample.id: K562_rep1 
assembly: hg19 
context: CpG 
resolution: base 
```

How come the row number in the object is different from the row number of the input file?
Or does methRead apply filtering on the input?

I am following Dave Tang's blog but he got the same row number as the input file. [https://davetang.org/muse/2013/05/09/using-encode-methylation-data/]

Thank you for your help!

Best regards,
Yichao

**Comment by alexg9010** [comment]
Hi @YichaoCaiDBS,

When I checked the code at Dave Tang's blog I saw that he was actually using version 0.92 and I found a release close to that rather old version on github [here](https://github.com/al2na/methylKit/tree/v0.9.5). 
The old `read` function that he used was now replaced with `methRead`, but as you already suspected `methRead` applies a  coverage filter using the `mincov` argument (default: 10), which leads to the different row numbers. 

**TL;DR**

I guess you could keep all entries of the rrbs file by setting `mincov` to 0, but for an actual analysis I suggest you keep the cutoff 10, since a methylation score derived from less than 10 reads is imho rather unreliable. 

If you have any other question relating to the package please also consider using our other channels to [stay up to date](https://github.com/al2na/methylKit#staying-up-to-date).

Best,

Alex
